### ISSUE TITLE
Cannot access any images but first in gallery with keyboard navigation

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#11157]
**Describe the bug**
Keyboard navigation with image gallery needs improvement. All images beyond the first are inaccessible.

**To Reproduce**
1. Insert a gallery with at least 2 images
2. Put focus into preceding block
2. Hit `TAB` repeatedly
3. You'll pass through the block inserter, block controls, remove image button, and first image caption before moving to the "Upload an Image" button. You will not go to any subsequent image remove buttons or captions.

**Expected behavior**
All images in a gallery should be accessible via keyboard

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser Firefox 63

**Additional context**
- WordPress 5.0-beta1-43823, Twenty Nineteen, no plugins

**Comment by Smith-erin** [comment]
Could not reproduce this error.

OSX 10.14, Chrome 69, WordPress 5.0-beta1-43823, Twenty Nineteen, no plugins

**Comment by earnjam** [comment]
Duplicate of #3836
